src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -26,12 +25,12 @@ import { isInterrupted } from "../../utils/pause";
 import { makeLocationId } from "../../utils/breakpoint";
 import showContextMenu from "./BreakpointsContextMenu";
 
-import type { Breakpoint, Location } from "../../types";
+import type { Breakpoint, Location, Source } from "../../types";
 
 import "./Breakpoints.css";
 
-type LocalBreakpoint = Breakpoint & {
-  location: any,
+export type LocalBreakpoint = Breakpoint & {
+  location: Location & { source: Source },
```

Comment:
could we avoid updating Location if we make this `Location & { source?: Source },`